libraries/rig-package/README.md

```diff
@@ -0,0 +1,199 @@
+
+If **project1** and **project2** are regular libraries, then their **api-extractor.json** now reduces to this:
+
+**project1/config/api-extractor.json**
+```js
+{
+  "$schema": "https://developer.microsoft.com/json-schemas/api-extractor/v7/api-extractor.schema.json",
+  "extends": "example-rig/profile/library/api-extractor.json"
+}
+```
+
+Whereas if **project3** is an SDK library, then it might look like this:
+
+**project3/config/api-extractor.json**
+```js
+{
+  "$schema": "https://developer.microsoft.com/json-schemas/api-extractor/v7/api-extractor.schema.json",
+  "extends": "example-rig/profile/sdk-library/api-extractor.json"
+}
+```
+
+Using `"extends"` definitely made the config file shorter!  But imagine that we have a large monorepo with 100 projects.
+And each project has 5 config files like **api-extactor.json**.  We still have to copy+paste 100 x 5 = 500 config files
+across all our project folders.
+
+Can we do better?
+
+
+## rig.json eliminates files entirely
+
+The idea is to replace `config/api-extractor.json` and `config/other-tool.json` (and any other such files)
+with a single file `config/rig.json` that delegates everything to the rig package:
+
+**project3/config/rig.json**
+```js
+{
+  "$schema": "https://developer.microsoft.com/json-schemas/rig-package/rig.schema.json",
+
+  "rigPackageName": "example-rig",
+  "rigProfile": "sdk-library"
+}
+```
+
+This eliminates the `"extends"` stub files entirely.  A tool that implements the `rig.json` system would
+look for its config file (`config/<targetFile>.json`) using this procedure:
+
+1. First check for `config/<targetFile>.json` in the project folder; if found, use that file.  OTHERWISE...
+2. Check for `config/rig.json`; if found, then this project is using a rig package.  Read the `<rigPackageName>`
+   and `<rigProfile>` settings from that file.
+3. Use Node.js module resolution to find the `<rigPackageName>` package folder (let's call that `<rigPackageFolder>`)
+4. Check for `<rigPackageFolder>/profile/<rigProfile>/<targetFile>.json`; if found, use that file.  OTHERWISE...
+5. If the `<targetFile>.json` cannot be found in either of these places, the behavior is left to the tool.
+   For example, it could report an error, or proceed using defaults.
+
+This eliminates the need for most config files that support this system.  In cases where we need a project-specific
+customization, the `"extends"` field is still supported.  For example, **project1** can still add a custom setting
+like this:
+
+**project1/config/api-extractor.json**
+```js
+{
+  "$schema": "https://developer.microsoft.com/json-schemas/api-extractor/v7/api-extractor.schema.json",
+  "extends": "example-rig/profile/library/api-extractor.json",
+
+  // Custom setting:
+  "mainEntryPointFilePath": "<projectFolder>/lib/custom.d.ts",
+}
+```
+
+The result is a much shorter inventory of files:
+
+```
+example-rig/package.json
+example-rig/profile/library/api-extractor.json
```

Comment:
`library` seems random here. I think it'd be easier to wrap your bran around if it was also `config`. That way the config files in the rig package are in the same place as they would be in a consuming project.